Website Optimization/Testing and analyzing.md

```diff
@@ -0,0 +1,101 @@
+The whole purpose of website optimization is to make your site as performant to search engines and people as possible. The first step to achieving that goal is to identify issues - that’s the foundation of any great optimization process. You can find many tools in the market specialized just for that. Because so many available options exist, we covered the most used and reliable ones. Before diving deep into testing, let's cover metrics and their definitions which are important:
+
+**First visit** - the time it takes for the page to load when there are no resources in the browser cache
+
+**Repeat visit** - the time it takes for the page to load when there are some resources present in the browser cache
+
+**Load Time** - the time it takes for the page to be completely downloaded, rendered, and then displayed to the user
+
+**Time to First Byte (TTFB)** - the time it takes from the start of the request to when the browser receives the first byte of the response from the server (best if 600ms or less)
+
+**First Contentful Paint (FCP)** - the time it takes for the browser to display the page content (best if 0.9s or less)
+
+**Time to Interactive (TTI)** - the time it takes for the page to become fully interactive (best if 2.5s or less)
+
+**Speed Index** - how quickly the contents of the page are visibly populated, depends on the browser viewport size (best if 1.3s or less)
+
+**Total Blocking Time (TBT)** - the time the page was blocked from interaction during the loading process (best if 150ms or less)
+
+**Largest Contentful Paint (LCP)** - the time for the largest content element to be displayed on the page (best if 1.2s or less)
+
+**Cumulative Layout Shift (CLS)** - an amount of layout shifting as page loads (best if 0.1 or less)
+
+These are factors almost all tools provide in reports. For ideal performance, the recommended score of each metric should be targeted.
+
+### Lighthouse
+
+Lighthouse is an open-source tool for testing web pages with audits for performance, accessibility, progressive web apps, SEO, and more. It can be run from Chrome DevTools, command line, node package, or WebUI.
```

Comment:
```suggestion
Lighthouse is an open-source tool for testing web pages with audits for performance, accessibility, progressive web apps, SEO, and more. It can be run from your browser's DevTools (in Chrome and Edge, and probably other Chromium browsers), command line, node package, or WebUI.
```